app/event_handlers/claim_export_handler.rb

```diff
@@ -1,6 +1,8 @@
 class ClaimExportHandler
   def handle(claim)
-    Export.claims.create resource: claim
+    ExternalSystem.containing_office_code(claim.office_code).each do |system|
```

Comment:
Both this export handler and the response export handler below have had the same change which simply does the same as before but for each external system that should be sent this data.